tests/uvdata/test_mwa_corr_fits.py

```diff
@@ -1337,4 +1344,4 @@ def test_van_vleck(benchmark, cheby):
         good_ants = np.delete(np.unique(uv2.ant_1_array), 76)
         uv2.select(antenna_nums=good_ants)
 
-    assert np.allclose(uv1.data_array, uv2.data_array)
+    assert uv1._data_array == uv2._data_array
```

Comment:
this too

Reply:
ok, I got that one and another one in that file plus a few more I found in other places. I swear I searched the whole repo before but I guess I used a slightly different search this time.